src/create-with-theme.js

```diff
@@ -1,31 +1,26 @@
 const React = require('react');
-const PropTypes = require('prop-types');
 const channel = require('./channel');
+const createThemeListener = require('./create-theme-listener');
 
 function createWithTheme(CHANNEL = channel) {
+  const themeListener = createThemeListener(CHANNEL);
   return Component =>
     class withTheme extends React.Component {
```

Comment:
Well its not a normal function, its a class, but if we think of it as of functional component, which it isn't, then we could also use lower case. In any case I didn't think of how it is displayed in the dev tools. I am not sure whats the common practice is. In react-jss we always used upper cased version: https://github.com/cssinjs/react-jss/blob/master/src/createHoc.js#L60 ... lets look up in the most common things like redux or something.